### ISSUE TITLE
Brems cross section default

### ISSUE THREAD

**Issue by rtownson** [nrc-cnrc/EGSnrc#1209]
Reminder: Fred just mentioned the default brems cs is BH, but more accurate is NRC. I think we already changed this default to be NRC, but we should double check.

**Comment by ftessier** [comment]
Yes, and also turn on electron-impact ionization by default, so that EGSnrc is most accurate out of the box. Also consider including Compton radiative corrections by default?

**Comment by ftessier** [comment]
By default, the brems sampling is option is `BH`, according to [`egsnrc.macros`](https://github.com/nrc-cnrc/EGSnrc/blob/master/HEN_HOUSE/src/egsnrc.macros#L2276). A review of default parameters is in order, especially since some applications override the defaults

Default parameters ought to be the same and the most exact  across the board, with documentation on possible efficiency improvement using alternative parameters in each application.
